hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/standalone/TransactionExecutorsTest.java

```diff
@@ -338,4 +348,27 @@ private Bytes resourceAsBytes(@NonNull final String loc) {
             throw new UncheckedIOException(e);
         }
     }
+
+    public static X509Certificate randomX509Certificate() {
```

Comment:
Can this test not use the method in `RosterRetrieverTest`?

Reply:
Not accessible, so had to copy it.